src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 from pathlib import Path
+from turtle import st
```

Comment:
```suggestion
```
stupid vs code extensions

Reply:
What does this one do? : O

Reply:
I believe that is the command to show a turtle. Haha. What an Easter egg. Some goes to read an NWB file and a turtle pops up on their screen....